spec.md

```diff
@@ -615,6 +621,24 @@ message PublishVolumeInfo {
   // possible.
   map<string, string> values = 1;
 }
+
+message Principal {
+  // describes attributes for defining the principal identity of
+  // a volume binding for a container.
+
+  // This field is OPTIONAL. If specified it contains the CO namespace
+  // identity in which the the following identity information (user /
+  // group) exists.
+  string namespace = 1;
+  // This field is OPTIONAL. If specified it contains the CO user
+  // identity holding ownership of a volume, similar to the user_id
+  // in a POSIX file system.
+  string user = 2;
+  // This field is OPTIONAL. If specified it contains the CO group
+  // identities holding ownership of a volume, similar to the group_id
+  // in a POSIX file system.
+  repeated string group = 3;
+}
```

Comment:
ACL might be a bit over the top. As the user and groups settings inside a container are completely arbitrary we need to pin the identity with which the given containers operations access the volume and it's content. All parties from that container use the same identity and access mode. Other Containers can access the same volume with different settings. That's why we also need the publishing on a per container and not per volume based (see #40 ).